extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -23,26 +24,367 @@ describes.realWin('amp-autocomplete', {
   },
 }, env => {
 
-  let win;
-  let element;
+  let win, doc;
```

Comment:
These tests are really great! They're super thorough and based around the behaviors of your component. My one suggestion is that they are pretty deeply nested, so consider ways you can split up this file into two or more test files, each containing tests that center around a common theme.

Reply:
I separated this roughly in half into build-related (`test-amp-autocomplete.js`) and everything else (`test-amp-autocomplete-unit.js`). Let me know if you imagined a different way to separate these tests or if you think of a better way to name the two files, since they are both technically unit tests.

Reply:
Cool, I think that's a sensible way to divide the tests. Hmm naming is hard. Maybe `test-amp-autocomplete-init.js` for the build related, and `test-amp-autocomplete.js` for general tests? Thanks for separating those